src/test/java/org/broadinstitute/hellbender/tools/sv/cluster/CanonicalSVCollapserUnitTest.java

```diff
@@ -1128,16 +1109,32 @@ public Object[][] collapseSampleGenotypesTestData() {
                         ),
                         Lists.newArrayList(
                                 createGenotypeTestAttributesWithCNQ(2, 2, 30),
-                                createGenotypeTestAttributesWithCNQ(2, 1, 30)
+                                createGenotypeTestAttributesWithCNQ(2, 3, 30)
                         ),
                         Allele.REF_N,
                         GenotypeBuilder.create(
                                 "sample",
                                 Lists.newArrayList(Allele.NO_CALL, Allele.NO_CALL),
+                                createGenotypeTestAttributesWithCNQ(2, 2, 30)
+                        )
+                },
+                {
+                        "sample",
+                        Lists.newArrayList(
+                                Lists.newArrayList(Allele.NO_CALL, Allele.NO_CALL),
+                                Lists.newArrayList(Allele.NO_CALL, Allele.NO_CALL)
+                        ),
+                        Lists.newArrayList(
+                                createGenotypeTestAttributesWithCNQ(2, 2, 30),
                                 createGenotypeTestAttributesWithCNQ(2, 1, 30)
+                        ),
+                        Allele.REF_N,
+                        GenotypeBuilder.create(
+                                "sample",
+                                Lists.newArrayList(Allele.NO_CALL, Allele.NO_CALL),
+                                createGenotypeTestAttributesWithCNQ(2, 2, 30)
```

Comment:
So we prefer a hom ref CN for MCNVs but non-ref for biallelic sites? That seems a bit inconsistent

Reply:
True is is inconsistent. I think one could argue for preferring the copy states farther from neutral, which would make the copy state distribution wider. Here I've chosen to, after considering RD quality, prefer copy states closer to neutral. This is just a more conservative prior. I don't think it makes sense, however, to have both het>hom-ref and het>hom-var for mCNVs since they are common by definition such that losing variants is not as problematic as for rare variants (which constitute most of the call set). 

There are other technical reasons for wanting to prefer het over hom-ref with non-depth-only calls as well (e.g. breakpoint shifting on SR signatures), whereas mCNVs are usually depth-only.